x/blob/client/cli/wirepayfordata.go

```diff
@@ -37,14 +37,20 @@ func CmdWirePayForBlob() *cobra.Command {
 			}
 
 			// decode the blob
-			blob, err := hex.DecodeString(args[1])
+			rawBlob, err := hex.DecodeString(args[1])
 			if err != nil {
 				return fmt.Errorf("failure to decode hex blob: %w", err)
 			}
 
+			// TODO: allow for more than one blob to be sumbmitted via the cliw
```

Comment:
```suggestion
			// TODO: allow for more than one blob to be submitted via the cli
```

Reply:
Also should we change the name of this file?

Reply:
https://github.com/celestiaorg/celestia-app/issues/1138